lib/service/storage/elasticsearch.js

```diff
@@ -210,6 +210,54 @@ class ElasticSearch extends Service {
       .catch(error => this._esWrapper.reject(error));
   }
 
+  /**
+   * Returns detailed multi-level storage stats data
+   *
+   * @returns {Promise.<Object>}
+   */
+  async stats () {
+    const esRequest = {
+      metric: ['docs', 'store'],
+    };
+
+    const { body } = await this._client.indices.stats(esRequest);
+
+    const response = {
+      indexes: {},
+      size_in_bytes: 0,
+    };
+
+    for (const [esIndex, indexInfo] of Object.entries(body.indices)) {
```

Comment:
Nitpicking/naming: the name `indexInfo` may lead to confusion with Kuzzle indexes
```suggestion
    for (const [esIndex, esIndexInfo] of Object.entries(body.indices)) {
```